src/App.jsx

```diff
@@ -1,17 +1,25 @@
-import React, { useState } from 'react';
-import Data from './Data.json';
-
+import React, { useState, useCallback, useEffect } from 'react';
+import { v4 } from 'uuid';
 import Header from './Header';
 import ToDoList from './ToDoList';
 import ToDoForm from './ToDoForm';
 
 function App() {
-    const [toDoList, setToDoList] = useState(Data);
-    const [listId, setListId] = useState(toDoList.length);
+    return (
+        <div>
+            <div className="App">
+                <Header />
+                <Todos />
+            </div>
+        </div>
+    );
+}
+function Todos() {
+    const [toDoList, setToDoList] = useState(null);
 
-    const handleButton = (id) => {
+    const onRemoveTodo = ({ id }) => {
```

Comment:
i suppose `{ id }` is the deconstruction of `task` paramater but it is not very clear. How about naming it taskId or not deconstructing at all?

Also, let's choose one - "task" or "todo" and align all related namings accordingly